.github/workflows/continuous-integration.yml

```
@@ -105,6 +105,24 @@ jobs:
           git config --global --add safe.directory "/var/www/glpi/plugins/${{ inputs.plugin-key }}"
           echo -e "\033[0;33mExecuting Composer install...\033[0m"
           composer install --ansi --no-interaction --no-progress --prefer-dist
+      - name: "Check for common dependencies"
+        if: ${{ !cancelled() && hashFiles(format('{0}/composer.json', inputs.plugin-key)) != '' }}
+        run: |
+          echo -e "\033[0;33mChecking for common dependencies between plugin and core...\033[0m"
+          sudo apt-get install -y jq
+          PLUGIN_DEPS=$(composer show --format=json 2>/dev/null | jq -r '.installed[] | select(.["direct-dependency"] == true and (.abandoned == false or .abandoned == null)) | .name' | grep -v '^glpi-project/tools$')
+          CORE_DEPS=$(cd ../../ && composer show --format=json 2>/dev/null | jq -r '.installed[] | select(.["direct-dependency"] == true and (.abandoned == false or .abandoned == null)) | .name')
```

Maybe it can be simplified with the `--direct` parameter.

```suggestion
          PLUGIN_DEPS=$(composer show --direct --format=json 2>/dev/null | jq -r '.installed[] | .name' | grep -v '^glpi-project/tools$')
          CORE_DEPS=$(cd ../../ && composer show --direct --format=json 2>/dev/null | jq -r '.installed[] | .name')
```

Why is `2>/dev/null` required?